_posts/2017-03-31-introducing-deckgl-v4.md

```diff
@@ -9,21 +9,21 @@ tags:
 
 ## Introducing deck.gl v4.0
 
-We've just released a new major version of deck.gl, so this is a good time to share some information about the improvements that went into the new version, the rationale behind them, and provide some hints on what we are currently thinking about the future of deck.gl.
+We've just released a new major version of deck.gl, so this is a good time to share some information about the improvements that went into the new version the rationale behind them, and provide some hints on what we are currently thinking about the future of deck.gl.
 
 ### What Motivated the Release?
 
 When we did our first [external announcement](https://eng.uber.com/deck-gl-framework/) of deck.gl in November 2016 (announcing the deck.gl v3 release), we also stated our intention to improve and expand deck.gl's layer catalog in future releases, and that is a major part of what we are doing now through the deck.gl v4 release.
 
-deck.gl v3 saw rapid adoption across a number of internal data visualization applications here at Uber, and it didn't take long before an impressive list of new layers had been developed to support these applications. The most general of these layers soon started to be reused between applications, and it quickly became clear that including the "best" of these layers in a new official deck.gl release would likely be of high value both to internal and external developers.
+deck.gl v3 saw rapid adoption across a number of internal data visualization applications here at Uber, and it didn't take long before an impressive list of new layers had been developed to support these applications. For layers are reused between multiple applications, it became clear that open sourcing them in a new official deck.gl release would likely to benefit both to internal and external developers.
```

Comment:
> For layers ^ are reused between multiple applications

Missing `that` (layers that are ...)

Maybe say "for layers that `were`" to match the tense of the previous sentences?